modules/lead/toolchain/artifactory-xray.tf

```diff
@@ -0,0 +1,38 @@
+resource "random_string" "artifactory_xray_db_password" {
+  length  = 10
+  special = false
+ }
+ resource "random_string" "artifactory_xray_mongo_db_password" {
+   length  = 10
+   special = false
+}
+
+//  data "helm_repository" "jfrog" {
+//   name = "jfrog"
+//   url  = "https://charts.jfrog.io"
+// }
+
+resource "helm_release" "xray" {
```

Comment:
need to add a setting:

`server.service.type=ClusterIP`

server.service.type | Xray server service type | LoadBalancer
-- | -- | --